### PR TITLE
Updated code with Cloak Shredder and AIOI

### PR DESCRIPTION
**Pull request by rsk97**
There are some extra config files that we can remove later (currently using them for running multiple expts)

### CODE REVIEW

File: src/configs/cloak_2.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (tremblerz) [inline src/configs/cloak_2.json]: remove please

File: src/configs/cloak_3.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (tremblerz) [inline src/configs/cloak_3.json]: remove please

File: src/configs/deep_obfuscator.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
```
Comment (tremblerz) [inline src/configs/deep_obfuscator.json]: remove please

File: src/configs/deep_obfuscator_2.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (tremblerz) [inline src/configs/deep_obfuscator_2.json:1]: remove please

File: src/configs/split_infer_cifar.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (tremblerz) [inline src/configs/split_infer_cifar.json:1]: remove please

File: src/configs/split_inference.json [hunk 6]
```diff
@@ -10,7 +10,7 @@
     "dataset": "fairface",
```
Comment (tremblerz) [inline src/configs/split_inference.json:10]: remove please

File: src/configs/system_config_4.json [hunk 7]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (tremblerz) [inline src/configs/system_config_4.json]: Actually you can remove all configs except the ones belonging to new algos

File: src/main.py [hunk 8]
```diff
@@ -9,8 +9,13 @@
                     help='filepath for benchmark config, default: {}'.format(b_default))
```
Comment (tremblerz) [inline src/main.py]: prints not needed

File: src/configs/deep_obfuscator.json (hunk 3 above)
Comment (tremblerz) [reply src/configs/deep_obfuscator.json]: Can you remove this too? Thanks!

File: src/configs/split_infer_cifar.json (hunk 5 above)
Comment (tremblerz) [inline src/configs/split_infer_cifar.json]: This file too

File: src/configs/split_inference.json [hunk 9]
```diff
@@ -10,7 +10,7 @@
...
     "protected_attribute": "data",
     "prediction_attribute": "gender",
-    "exp_id": "1",
```
Comment (tremblerz) [inline src/configs/split_inference.json]: This too

File: src/configs/system_config.json [hunk 10]
```diff
@@ -1,5 +1,5 @@
 {
     "dataset_path": "/u/abhi24/Datasets/Faces/fairface/",
-    "experiments_folder": "/u/abhi24/Workspace/simba/experiments/",
```
Comment (tremblerz) [inline src/configs/system_config.json]: This too

File: src/interface.py [hunk 11]
```diff
@@ -1,3 +1,5 @@
+from torch.nn.modules.linear import Linear
```
Comment (tremblerz) [inline src/interface.py]: It seems this line is not needed?

File: src/main.py (hunk 8 above)
Comment (tremblerz) [reply src/main.py]: Can you remove these prints?

**Review by tremblerz — CHANGES_REQUESTED** [review]
We are almost set with this PR if you can remove the items I highlighted. Thanks for all your efforts, it is going to be a good bump for our repository

File: src/configs/system_config.json (hunk 10 above)
Comment (rsk97) [reply src/configs/system_config.json]: Are you sure we should remove this system config file too? This is the only example here. Pls let me know! Thx!

**Pull request merged**